tasks/prd-retirement-planner.md

```
@@ -0,0 +1,206 @@
+- Calculate earliest possible retirement date based on current trajectory
+- Display projections including net worth growth, years to goal, and sustainable monthly income under 3%/4% withdrawal rules
+- Support multiple investment return scenarios (conservative/moderate/aggressive)
+- Allow saving and comparing up to 3 retirement scenarios side-by-side
+- Persist retirement plans in local SQLite database
+- Pre-populate starting net worth from the most recent balance sheet entry
+
+---
+
+## User Stories
+
+### US-001: Create retirement planner database models
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)